how-to-guides/e2e-tracking/scripts/e2e_tracking.py

```diff
@@ -1,8 +1,8 @@
-# ========================================== #
-# How to track models end-to-end on Neptune? #
-# ========================================== #
+# %% ========================================== #
```

Comment:
> `%%`

What is this notation? (Just curious whether it has any function)

Reply:
This lets some IDEs (like DataSpell and VSCode) execute code blocks in scripts individually, like notebook cells